semgrep_output_v1.atd

```diff
@@ -2334,7 +2334,9 @@ type function_call <python decorator="dataclass(frozen=True)"> = [
   | CallValidate of fpath
   | CallResolveDependencies of dependency_source list
   | CallDumpRulePartitions of dump_rule_partitions_params
-  | CallTransitiveReachabilityFilter of transitive_finding list
+  (* for now, the transitive reachability filter takes only a single dependency graph as input.
+   * it is up to the caller to call it several times, one for each subproject *)
+  | CallTransitiveReachabilityFilter of ((found_dependency * downloaded_dependency list) * transitive_finding list)
```

Comment:
I think you want (found _dependency * downloaded_dependency) list no ?
Also why you talk about graph in the comment?

Reply:
correct, fixed.

The `(found _dependency * downloaded_dependency) list` encodes the dependency graph, so the comment is referring to that